inst/lintr/linters.csv

```diff
@@ -66,6 +66,7 @@ semicolon_linter,style readability default configurable
 semicolon_terminator_linter,style readability deprecated configurable
 seq_linter,robustness efficiency consistency best_practices default
 single_quotes_linter,style consistency readability default
+sort_linter,style readability best_practices efficiency
```

Comment:
I believe `style` is reserved for things in the official Tidyverse guide, the other tags look fine though :)